test/unit/com/unifina/controller/api/KeyApiControllerSpec.groovy

```diff
@@ -24,6 +24,7 @@ class KeyApiControllerSpec extends ControllerSpecification {
 			username: "me@me.com",
 			password: "pwd",
 			name: "name",
+			lastLogin: new Date(),
```

Comment:
Can remove `lastLogin` if you set the default field value in `SecUser.groovy` (also elsewhere in this file)